features/FxA-89-devices-view/README.md

```diff
@@ -0,0 +1,410 @@
+should go to the following URL,
+which works for all OSes and all languages:
+
+* https://www.mozilla.org/firefox/new/?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices
+
+It should contain the UTM parameters as specified
+and should not contain any locale in the URL.
+
+The link to get Firefox for iOS:
+ 
+* https://app.adjust.com/2uo1qc?campaign=fxa-device-manager&adgroup=ios&creative=button&fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8 
+* The `fallback` parameter  will force the App Store
+  since default is Play Store if link is opened
+  on desktop instead of a mobile device.
+
+The link to get Firefox for Android:
+
+* https://app.adjust.com/2uo1qc?campaign=fxa-device-manager&adgroup=ios&creative=button
+* No `fallback` parameter is required
+  since the URL will open the Play Store
+  if it doesn’t recognize any mobile device.
+
+The link labelled "Missing or duplicate devices?"
+will link to a SUMO article:
+
+* https://support.mozilla.org/kb/fxa-managing-devices
+
+When a user is using desktop Firefox
+and has a web session with accounts.firefox.com
+but is not connected with the current browser,
+we can provide a "Connect..." button that links to:
+
+* /signin?service=sync&context=fx_desktop_v3"
+
+
+## Implementation Plan
+
+We will implement this feature in several phases,
+intending each phase to be independently shippable
+if we so choose.
+
+Since the primary goal for this feature is security,
+we will first focus on implementing the parts
+related to disconnecting devices.
+This will be followed by additional notifications
+and security checks to improve the disconnection experience,
+and finally by the new affordances
+for connecting additional devices.
+
+### Phase 1
+
+First we'll make sure
+that the core disconnecting-your-devices experience
+works well and emits the necessary metrics,
+building on the existing devices view code.
+
+Acceptance criteria:
+
+* [ ] When I am connected to Sync on Desktop Firefox,
+      my device list shows that device
+      with a desktop icon,
+      a plausible last-active timestamp,
+      and a "Disconnect..." button.
+* [ ] When I am connected to Sync from Fennec or iOS on a phone,
+      my device list shows that device
+      with a phone icon,
+      a plausible last-active timestamp,
+      and a "Disconnect..." button.
+* [ ] When I am connected to Sync from Fennec or iOS on a tablet,
+      my device list shows that device
+      with a tablet icon,
+      a plausible last-active timestamp,
+      and a "Disconnect..." button.
+* [ ] When I sign in to Firefox Accounts on the web
+      without connecting to sync,
+      this does not appear as an item
+      in the device list.
+* [ ] The devices view does not show
+      any messaging about connecting new devices.
+* [ ] Clicking the "Disconnect..." button for a device
+      causes a modal confirmation dialog to be displayed
```

Comment:
IIUC the "placeholder device records" stuff that @philbooth has been working on will make Fennec and iOS devices show up properly in this list, albeit with generic names.  There is also active work on shipping proper device registration on both platforms.  So I *think* they'll behave well enough to make this feature feel complete.  @vladikoff do you have a specific concern with these platforms?